schemas/dbt/manifest/v8.json

```diff
@@ -585,6 +585,10 @@
           "type": "boolean",
           "default": true
         },
+        "constraints_enabled": {
```

Comment:
How do you test the manifest?

Reply:
quis verificat ipsos verificantia? quis manifestat ipsos manifesta?

Reply:
google translate:
> Who verifies the verifiers? Who makes them manifest?